### DISCUSSION TITLE
Feature request: set default accommodations for individual users

### DISCUSSION THREAD

**Discussion by dlglin** [openwebwork/webwork2#1865] [Ideas]
We regularly have students who have an accommodation for timed exams (e.g. 50% extra time).  The idea is to add a feature where these accommodations can be recorded for an individual student, and every time a timed assessment is assigned to that user the time limit is automatically adjusted.

**Comment by dlglin** [comment]
From https://webwork.maa.org/moodle/mod/forum/discuss.php?d=8244 if this is implemented then these accommodations should be able to be imported from a class list file as well as managed through the UI.

**Comment by somiaj** [comment]
The new accommodation factor setting addresses this.
